### PR TITLE
Url executor redux

### LINKED ISSUE #11: Implement API executor
We should pull in some http library that provides statistics easily (that also lets us hit random endpoints) and wire in a executor

### PR DESCRIPTION
**Pull request by chrisbenincasa**
Really simple URL executor impl. We can figure out error handling, etc later, if you want. 

Closes #11

### CODE REVIEW

File: core/src/main/scala/io/paradoxical/aetr/core/config/readers/StandardTypeReaders.scala [hunk 1]
```diff
@@ -2,17 +2,10 @@ package io.paradoxical.aetr.core.config.readers
 
 import com.typesafe.config.Config
 import java.io.File
-import java.net.URL
 import net.ceedubs.ficus.readers.ValueReader
 import scala.util.matching.Regex
 
 trait StandardTypeReaders {
-  implicit val url = new ValueReader[URL] {
```
Comment (chrisbenincasa) [inline core/src/main/scala/io/paradoxical/aetr/core/config/readers/StandardTypeReaders.scala]: ficus provides this

File: core/src/main/resources/reference.conf [hunk 2]
```diff
@@ -1,4 +1,9 @@
 aetr {
+  meta = {
+    host = "http://localhost"
```
Comment (devshorts) [inline core/src/main/resources/reference.conf]: YES SO GOOD

File: core/src/main/resources/reference.conf [hunk 3]
```diff
@@ -1,4 +1,9 @@
...
+    complete_callback_path = "/api/v1/complete/:token"
```
Comment (devshorts) [inline core/src/main/resources/reference.conf]: Smart, this lets people put a reverse proxy and they can have the endpoint be whatever they want

Comment (chrisbenincasa) [reply core/src/main/resources/reference.conf]: 🎊 🔥

**Pull request merged**